README.md

```diff
@@ -2,43 +2,42 @@
 This application is used to build ElasticSearch indices in order to test different ontology files and the terms that are created from them. The application is provided a config file and a list of ontology files to run against. A local Elasticsearch container is spun up for queries to be run against and tested.
 
 ### Config
-The service builds indices based off of a config file that is provided to it.  The config contains the name of the index, the server host names minus http and port (the server name will be elastic if running against the local elastic docker container), and the list of files to index.
+The service builds indices based off of a config file that lives in `configs/config.json`.  The config contains the name of the index, the server host names minus http and port (the server name will be elastic if running against the local elastic docker container), and the list of files to index.
 
-If using files from your local machine, place them into the src/main/resources/ontologies/ folder and they will be included with the gradle build. When running, both the local and remote files are copied into the docker container running the app to be processed.
+Copy/download all files to your local machine. Place them into the src/main/resources/ontologies/ folder and they will be included with the gradle build. When running, all files are copied into the docker container running the app to be processed.
```

Comment:
I think we should automatically generate a config and ontology resources by perhaps providing a script which can be run which would pull the latest ontologies from DOID (for example). Then this script could simply be used by the `builder` layer and copied into the final container without any user intervention.

Reply:
I'd like to tackle something like this separately - the goal of this PR was to just get it up and running from its original non-functional state.